site/docs/components/accordion/accessibility.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Accordion
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<KeyboardControls>
+<KeyboardControl keyOrCombos="Tab">
```

Comment:
Where did we land on include Tab and Shift + Tab, I thought we weren't adding them

Reply:
most of the components have them. Should we be consistent, one way or the other?